tests/goal_tests.cpp

```diff
@@ -85,55 +85,55 @@ TEST_CASE("pick_ik::make_pose_cost_fn") {
         Eigen::Translation3d(0.0, 0.0, 0.0) * Eigen::AngleAxisd(2.0, Eigen::Vector3d::UnitY());
 
     SECTION("Goal is frame") {
-        auto const cost_fn = pick_ik::make_pose_cost_fn(zero_frame, 0, 0.0);
+        auto const cost_fn = pick_ik::make_pose_cost_fn(zero_frame, 0, 1.0, 0.0);
```

Comment:
Worth also adding the zero position scale paths in these unit tests? I think yes.